### PR TITLE
8.0 add XLS report to Aged Partner Balance

### PR DESCRIPTION
**Pull request by luc-demeyer**
This PR covers the following:
- adds xls export to the Aged Partner Balance report
- Fix so that reports are not broken on last and first day ofthe fiscal year
- a minor change to the trial balance xls report to facilitate customisation via inherit
- a couple of code style changes to align with the OCA guidelines

### CODE REVIEW

File: account_financial_report_webkit/wizard/__init__.py [hunk 1]
```diff
@@ -29,3 +8,6 @@
 from . import aged_open_invoices_wizard
 from . import print_journal
 from . import aged_partner_balance_wizard
+# uncomment import infra when running Odoo without
+# PR https://github.com/odoo/odoo/pull/14891
```
Comment (StefanRijnhart) [inline account_financial_report_webkit/wizard/__init__.py]: This upstream PR was merged. Can you remove the workaround code?

File: account_journal_report_xls/wizard/account_common_report_fix.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2009-2017 Noviat.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+"""
+Bypass of bug in Odoo:
+The financial reports do not work on first and last day of the Fiscal Year.
+This fix can be removed after merge of
+PR https://github.com/odoo/odoo/pull/14891.
```
Comment (StefanRijnhart) [inline account_journal_report_xls/wizard/account_common_report_fix.py]: Can be removed as well

**Comment by andreschenkels** [comment]
Hi Luc,
Can I help merging this stuff?

**Comment by hbrunn** [comment]
superseded by #352

**Pull request closed without merging**